### PR TITLE
Add `@husnuljahneer` as L1 Maintainer

### PR DESCRIPTION
**Pull request by husnuljahneer**

### CODE REVIEW

**Review by beerose — CHANGES_REQUESTED** [review]

**Comment by beerose** [comment]
Hey @husnuljahneer, any update on this PR?

File: README.md [hunk 1]
```diff
@@ -239,6 +239,14 @@ _Issue triage, pull request triage, community encouragement and moderation, etc_
...
       </a>
     </td>
+<td align="center">
+     <a href="https://jahneer.me">
+      <img src="https://avatars.githubusercontent.com/husnuljahneer" width="100px;" alt="@WarM4chineRoxX#2013 avatar" /><br />
+        <sub>
+          <b>Husnul Jahneer</b>
+        </sub>
+      </a>
+    </td>
```
[the remark this answers is not in the document]
Comment (husnuljahneer) [reply README.md]: I have done the formatting.

**Comment by beerose** [comment]
Could you also fix the conflicts? Also, it looks like you changed your github username — could you remove references to `@WarM4chineRoxX`?  And please resolve the other review comment 🙏

File: README.md [hunk 2]
```diff
@@ -752,6 +760,7 @@ Thanks to these wonderful people ([emoji key](https://allcontributors.org/docs/e
   </tr>
   <tr>
     <td align="center"><a href="https://saheen.codes"><img src="https://avatars.githubusercontent.com/u/2632896?v=4?s=100" width="100px;" alt=""/><br /><sub><b>saheenshoukath</b></sub></a><br /><a href="https://github.com/blitz-js/blitz/commits?author=shellord" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://jahneer.me"><img src="https://avatars.githubusercontent.com/husnuljahneer" width="100px;" alt=""/><br /><sub><b>Husnul Jahneer</b></sub></a><br /><a href="https://github.com/blitz-js/blitz/commits?author=husnuljahneer" title="Documentation">📖</a></td>
```
Comment (beerose) [inline README.md:763]: Please remove this part — it's generated and it will be automatically added after merging your first PR.

**Review by beerose — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (husnuljahneer) [reply README.md]: Done.

**Pull request merged**